server/src/main/java/org/folio/shared/index/api/SharedIndexService.java

```diff
@@ -85,44 +83,50 @@ static String getQueryParameter(RequestParameters params) {
     PgCqlQuery pgCqlQuery = getPqCqlQueryForRecords();
     RequestParameters params = ctx.get(ValidationHandler.REQUEST_CONTEXT_KEY);
     pgCqlQuery.parse(getQueryParameter(params));
-    String m = getParameterString(params.queryParameter("matchkeyid"));
     Storage storage = new Storage(ctx);
-    if (m != null) {
-      int maxIterations = getParameterInteger(
-          params.queryParameter("maxiterations"), MATCH_MAX_ITERATIONS);
-      List<String> matchKeyIds = Arrays.asList(m.split(","));
-      return storage.getCluster(pgCqlQuery.getWhereClause(), matchKeyIds, maxIterations)
-          .map(records -> {
-            JsonArray items = new JsonArray();
-            records.forEach((k, v) -> items.add(v));
-            JsonObject result = new JsonObject();
-            result.put("items", items);
-            result.put("totalRecords", records.size());
-            HttpResponse.responseJson(ctx, 200).end(result.encode());
-            return null;
-          });
-    }
     return storage.getSharedRecords(ctx, pgCqlQuery.getWhereClause(),
```

Comment:
Consider dropping the term "shared" records in favour of "global" records as in "getGlobalRecords" and "getGlobalRecordById"

Reply:
ok.. FWIW I prefer the bib record notion better, because global could mean a lot of things. And certainly from the VU point of view the cluster is kind of global.

Reply:
now renamed fwiw